ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -62,15 +65,24 @@ template <typename ShapeFunction, typename IntegrationMethod,
           unsigned GlobalDim>
 class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
 {
+    // Pressure always first
+    static const int pressure_index = 0;
+    static const int pressure_size = ShapeFunction::NPOINTS;
+    // per component
+    static const int concentration_size = ShapeFunction::NPOINTS;
```

Comment:
Please decide how to call a thing: component or concentration. Stick to it.
Maybe `single_component_size`.